### PR TITLE
add format_checker example to anonymization quickstart notebook

### PR DESCRIPTION
**Pull request by michaellevy**
In the format_checker example, I changed the expected format to what I want to propose to the telcos in Bangladesh: that all leading 0s and 880s and +s be stripped, and I had that example fail to show how the functionality works.

### CODE REVIEW

File: quickstart/documentation_anonymizer.ipynb [hunk 1]
```diff
@@ -1052,42 +1050,89 @@
...
-    "    if raw.startswith('+'):\n",
-    "        no_plus = raw.strip('+')\n",
-    "        if not no_plus.startswith('880'):\n",
-    "            return False\n",
-    "        if not no_plus[3:].startswith('01'):\n",
-    "            return False\n",
-    "    \n",
-    "    else:\n",
-    "        if not raw.startswith('001'):\n",
-    "            return False\n",
```
[the remark this answers is not in the document]
Comment (michaellevy) [reply quickstart/documentation_anonymizer.ipynb:1065]: i wouldn't think so. that seems like a US number being 520-911-1234, but i'll ask one of my colleagues in bangladesh for clarity on what we can expect.

File: quickstart/Untitled.ipynb
[the remark this answers is not in the document]
Comment (michaellevy) [reply quickstart/Untitled.ipynb:1]: no, will remove...

[the remark this answers is not in the document]
Comment (michaellevy) [reply quickstart/Untitled.ipynb:1]: good catch

**Comment by michaellevy** [comment]
Note that this is dependent on [these changes](https://github.com/Global-Policy-Lab/cider/pull/43) to Cider, which aren't yet merged into main.

**Review by lselker13 — APPROVED** [review]
LGTM! Thanks for doing this.

**Pull request merged**